l20n/migrate/__init__.py

```diff
@@ -0,0 +1,5 @@
+# coding=utf8
+
+from context import MergeContext
+from transforms import CONCAT, COPY, EXTERNAL, PLURALS, REPLACE, SOURCE
+from changesets import convert_blame_to_changesets
```

Comment:
Look at http://flake8.pycqa.org/en/latest/user/ignoring-errors.html#in-line-ignoring-errors, trailing those lines with `  # noqa: F401